t/Address.t

```diff
@@ -1,6 +1,7 @@
 use strict;
 use warnings;
 
+use Math::BigInt;
 use Math::Int128 qw(uint128);
```

Comment:
Ah, good point. There will be the same issue with the MaxMind::DB::Reader tests.